recipes/arm-gnu-toolchain/all/test_v1_package/conanfile.py

```diff
@@ -0,0 +1,31 @@
+from conans import ConanFile, CMake, CMakeToolchain
```

Comment:
There is no `CMakeToolchain` in `conans`. That might be trickier to get this working with a V1 test package. I think you can now drop the `test_v1_package` since it is after May 15th, see https://github.com/conan-io/conan-center-index/discussions/17406